tests/consumption_tests/test_linux_consumption.py

```diff
@@ -242,8 +244,25 @@ def test_opencensus_with_extensions_enabled(self):
             ctrl.assign_container(env={
                 "AzureWebJobsStorage": self._storage,
                 "SCM_RUN_FROM_PACKAGE": self._get_blob_url("Opencensus"),
-                "PYTHON_ENABLE_WORKER_EXTENSIONS": "1",
-                "AzureWebJobsFeatureFlags": "EnableWorkerIndexing"
+                PYTHON_ENABLE_WORKER_EXTENSIONS: "1"
+            })
+            req = Request('GET', f'{ctrl.url}/api/opencensus')
+            resp = ctrl.send_request(req)
+            self.assertEqual(resp.status_code, 200)
+
+    @skipIf(sys.version_info.minor != 10,
+            "This is testing only for python310")
```

Comment:
Why 3.10 only? is this due to the image limitations?

Reply:
I'm using an existing function app which is in 3.10. Only adding the indexing app setting and verifying if we get the same result.